mobile/src/libs/components/input-search/libs/constants/constants.ts

```diff
@@ -0,0 +1,6 @@
+const DEFAULT_SEARCH_PAYLOAD = {
+  search: '',
+  name: 'search' as const,
+};
+
+export { DEFAULT_SEARCH_PAYLOAD };
```

Comment:
Why do we need to create 'constants' folder? 
I am not saying that it's wrong, but there is no such folder for sign-up-form (constants.ts is placed directly in the 'libs' folder) and I didn't create this folder in my tasks also

Reply:
it's a good point, let's do it as everywhere without constants folder

Reply:
Done